app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
change line number 2-3 
`def home`
`end`

to one line
`def home; end`

Reply:
ok done :)